input/data/2009/2009A%26A...507..389A/tev-000060.yaml

```diff
@@ -0,0 +1,27 @@
+source_id: 60
+reference_id: 2009A&A...507..389A
+telescope: hess
+
+data:
+  livetime: 52.5h
+  significance: 9.5
+  excess: 450
+  n_on: 2353
+  n_off: 25633
+  alpha: 0.074
+
+pos:
+  ra: {val: 13h2m41s, err: 6s}
+  dec: {val: -63d49m1s, err: 41s}
+
+morph:
+  type: gauss
+  # No sigma given in the paper
+
+spec:
+  model:
+    type: pl
+    parameters:
+      norm: {val: 1.1, err: 0.1, err_sys: 0.2, scale: e-12, unit: cm-2 TeV-1 s-1}
```

Comment:
I think you have to write 1e-12 instead of e-12:
```
$ make.py all

Traceback (most recent call last):
  File "./make.py", line 142, in <module>
    cli()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "./make.py", line 136, in make_all
    ctx.invoke(make_catalog)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "./make.py", line 88, in make_catalog
    CatalogMaker(config).run()
  File "/Users/deil/code/gamma-cat/gammacat/catalog.py", line 561, in __init__
    self.sources = self.read_source_data(source_ids=source_ids)
  File "/Users/deil/code/gamma-cat/gammacat/catalog.py", line 614, in read_source_data
    sed=sed,
  File "/Users/deil/code/gamma-cat/gammacat/catalog.py", line 156, in from_inputs
    cls.fill_spectral_model_info(data, dsi)
  File "/Users/deil/code/gamma-cat/gammacat/catalog.py", line 300, in fill_spectral_model_info
    data['spec_pl_norm'] = spec_pars['norm'].get_or_default('val').to('cm-2 s-1 TeV-1')
  File "/Users/deil/code/gamma-cat/gammacat/modeling.py", line 54, in get_or_default
    result = result * self.scale * self.unit
TypeError: can't multiply sequence by non-int of type 'ScalarFloat'
```